Source/DataSources/BillboardVisualizer.js

```diff
@@ -50,6 +50,7 @@ define([
     var EntityData = function(entity) {
         this.entity = entity;
         this.billboard = undefined;
+        this.textureValue = undefined;
```

Comment:
I guess it's not important here but we have a common use case in 3D models and imagery were we create an image/canvas, upload it to the GPU, and then drop the reference to it.  This is why the texture atlas resize is done by basically copying the previous texture (via rendering it to a framebuffer) - so the texture atlas does not need to keep a reference to all the images in system memory.